### PR TITLE
[WIP] - adds project page for Alaska Medicaid project

### PR DESCRIPTION
**Pull request by hannahkane**
Fixes issue(s) #2904.

[CircleCI](https://circleci.com/gh/18F/18f.gsa.gov/tree/alaska-dhss)

[:sunglasses: PREVIEW](https://federalist-proxy.app.cloud.gov/preview/18f/18f.gsa.gov/alaska-dhss/what-we-deliver/alaska-dhss/)

[Preview README for this branch](https://github.com/18F/18f.gsa.gov/blob/alaska-dhss/README.md)

Changes proposed in this pull request:
- add project page for Alaska
- add image for Alaska page

Checklist:
- [ ] All images being added have been optimized **_--> [learn more](18f.gsa.gov/styleguide/images/#using-and-optimizing-jpg-and-png) about how to optimize images <--_**

/cc @relevant-people

### CODE REVIEW

**Comment by hannahkane** [comment]
@awfrancisco - I'm calling this a WIP because I'm not sure if there are more files I need to edit before this will work. Can you please let me know, and otherwise review this PR? Thanks!

File: _services_projects/alaska-dhss.md [hunk 1]
```diff
@@ -3,6 +3,7 @@ agency: Alaska Department of Health and Social Services
 title: Modernizing Access to Healthcare
 subtitle: Modernizing public benefit eligibility systems
 permalink: /what-we-deliver/alaska-dhss/
+redirect_from: /project/alaska-dhss/
```
Comment (hannahkane) [inline _services_projects/alaska-dhss.md]: I don't think this is the right redirect (got this from the gDoc). If I do a redirect from https://modularcontracting.18f.gov/projects/ak-medicaid/, will it work, or does it only work if it's redirecting from within this domain?

Comment (awfrancisco) [reply _services_projects/alaska-dhss.md]: Yeah, this redirect won't work yet, but we'll leave it in for when we work on all the redirects for the modular contracting site.

**Comment by awfrancisco** [comment]
This pull request is good to go from my end. We've removed the testimonial from the other page. Any other work/changes need to be done?

**Comment by hannahkane** [comment]
@awfrancisco - no other changes needed from my perspective

**Pull request merged**